packages/test/test-end-to-end-tests/src/test/summarizerWithLocalChanges.spec.ts

```diff
@@ -41,19 +45,23 @@ class TestDataObject1 extends DataObject {
 
 	private readonly datastoreKey = "TestDataObject2";
 
-	public createdDataStoreId?: string;
-
 	protected async hasInitialized() {
```

Comment:
Can you put a comment on this class explaining the behavior, since it's a critical part of the tests below?

Reply:
added comments.